src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -15,6 +15,7 @@ import HookStore from 'app/stores/hookStore';
 
 const INTEGRATIONS_ANALYTICS_SESSION_KEY = 'INTEGRATION_ANALYTICS_SESSION' as const;
 const SORT_INTEGRATIONS_BY_WEIGHT = 'SORT_INTEGRATIONS_BY_WEIGHT' as const;
+const SHOW_INTEGRATION_DIRECTORY_CATEGORY_SELECT = 'SHOW_INTEGRATION_DIRECTORY_CATEGORY_SELECT' as const;
```

Comment:
I don't think you need `as const` on strings, since the variable is already a `const`, it can't be changed.

Reply:
I feel like I may have suggested this at some point, I just realized this recently lol